coalib/misc/CachingUtilities.py

```diff
@@ -0,0 +1,133 @@
+import calendar
+import os
+import pickle
+import time
+
+from coalib.output.Tagging import get_tags_dir as get_pickle_dir
```

Comment:
if the name is misfitting, the original function should rather get a more generic name that fits all uses